dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/repositories/DataRequestRepository.kt

```diff
@@ -36,7 +39,7 @@ interface DataRequestRepository : JpaRepository<DataRequestEntity, String> {
      * It also filters these results based on the provided identifier value and frameworks.
      * @param identifierValue to check for
      * @param reportingPeriod to check for
-     * @param dataTypes to check for
+     * @param dataTypeNames to check for
      * @returns the aggregated data requests
      */
```

Comment:
Parameter `requestStatus` is not part of the documentation

Reply:
Done